tools/zbpanidconflictflood

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+import pdb
+import sys
+import argparse
+from time import sleep
+import threading
+import logging
+logging.getLogger("scapy.runtime").setLevel(logging.ERROR) # suppress annoying Scapy IPv6 warning
+from killerbee import *
+from scapy.all import *
+
+# Jacked from killerbee project.
+def randmac(length=8):
```

Comment:
Any chance we could put randmac into the kbutils or some similar location in the main project so it doesn't have to live in multiple scripts?